src/colorizer/CanvasOverlay.ts

```diff
@@ -123,62 +134,107 @@ export default class CanvasWithOverlay extends ColorizeCanvas {
 
   // Wrapped ColorizeCanvas functions ///////
 
+  public get resolution(): Vector2 {
+    return this.innerCanvasSize.clone();
+  }
+
+  public getScaleInfo(): CanvasScaleInfo {
+    return this.innerCanvas.getScaleInfo();
+  }
```

Comment:
any reason for one of these to be a getter and one of these to be a function?

Reply:
Not particularly... I'll change `scaleInfo` to be a getter